### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by matiasgarcia91**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,11 +11,16 @@
...
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "express-graphql": "^0.9.0",
     "graphql": "^14.6.0",
+    "graphql-upload": "^10.0.0",
+    "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.15",
-    "mongoose": "^5.9.3"
+    "mongoose": "^5.9.3",
+    "react-native-dotenv": "^0.2.0"
```
Comment (matiasgarcia91) [inline package.json:23]: don't think you need this dependency here right?

File: schema/schema.js [hunk 2]
```diff
@@ -8,107 +15,265 @@ const {
...
+    },
+    login: {
+      type: UserType,
+      args: {
+        email: { type: new GraphQLNonNull(GraphQLString) },
+        password: { type: new GraphQLNonNull(GraphQLString) }
+      },
+      async resolve(parent, args) {
+        const user = await User.find({ email: args.email });
+        if (user.length < 1) {
```
Comment (matiasgarcia91) [inline schema/schema.js]: use `.findOne` and check just `if(!user)`

File: schema/schema.js [hunk 3]
```diff
@@ -8,107 +15,265 @@ const {
...
+          throw new Error("Could not find user");
+        } else {
+          const { id, name, email, country, jobless } = user[0];
+          const valid = await compare(args.password, user[0].password);
```
Comment (matiasgarcia91) [inline schema/schema.js]: then you don't get this array index which looks out of place

**Review by matiasgarcia91 — COMMENTED** [review]
Not much to comment, really good job man! I'd remark to remove the commented out code but i know you are still working so remember to do that once you finish. Also the `auth.js` file if you are going to handle that through GraphQL. 
I didn't know you were using Mongo as well, you really took on all the new stuff you could! 🥇 

With this and a regular React/Redux app your GH is going to be really neat to check out

File: package.json (hunk 1 above)
Comment (torkvell) [reply package.json:23]: You have the eyes of an eagle! Yes, not needed. Used it before but then removed it

**Pull request closed without merging**